components/NavigationDrawer/tests/unit/MDCNavigationDrawerScrollViewTests.m

```diff
@@ -21,7 +21,8 @@ @interface MDCBottomDrawerContainerViewController (ScrollViewTests)
 
 @property(nonatomic) BOOL scrollViewObserved;
 @property(nonatomic, readonly) UIScrollView *scrollView;
-@property(nonatomic) BOOL currentlyFullScreen;
+@property(nonatomic, readonly) CGFloat topHeaderHeight;
```

Comment:
_Tip:_ These appear to be unused. In that case, they should be removed.

Reply:
In the test it's `fakeBottomDrawer.topHeaderHeight` that property has to be exposed via the category.

Reply:
Oops, I misread where these properties were declared. You're right.